src/include/sof/audio/codec_adapter/codec/generic.h

```diff
@@ -185,6 +185,7 @@ struct comp_data {
 	struct comp_buffer *ca_source;
 	struct sof_ipc_stream_params stream_params;
 	uint32_t period_bytes; /** pipeline period bytes */
+	uint32_t deep_buff_bytes; /**< copy start threshold */
```

Comment:
Seems this would be better called copy_start_threshold ? since the name is not reflecting the comment.